libkbfs/data_types.go

```diff
@@ -364,8 +365,35 @@ const (
 	// folder.  Set to the empty string so that the default will be
 	// the master branch.
 	MasterBranch BranchName = ""
+
+	branchRevPrefix = "rev="
 )
 
+// MakeRevBranchName returns a branch name specifying an archive
+// branch pinned to the given revision number.
+func MakeRevBranchName(rev kbfsmd.Revision) BranchName {
+	return BranchName(fmt.Sprintf("%s%d", branchRevPrefix, rev))
```

Comment:
This is pretty straightforward; perhaps just use `strconv` here as it's cheaper?